wandb/sdk/wandb_feature.py

```diff
@@ -0,0 +1,65 @@
+#
+"""Feature Flags Module
+
+This module implements a feature flag system for the wandb library to enable experimental features
+and notify the user when features have been deprecated.
+
+Example:
+    import wandb
+    wandb.use_feature("wandb-service:beta")
+    wandb.use_feature("incremental-artifacts:beta,newfeature")
+"""
+
+import wandb
+from wandb.errors import UseFeatureError
+
+if wandb.TYPE_CHECKING:
+    from typing import List
+
+
+class _Features(object):
+    """Internal feature class"""
+
+    _features: List[str]
+
+    def __init__(self, features: str) -> None:
+        self._features = features.split(",")
+
+    def apply(self) -> None:
```

Comment:
Might be good to add some logic to handle improperly formatted feature names.

Reply:
We can, but it is handled, it wont match a lookup, worst that can happen is amessage saying feature KF*#@HLFA isnt supported.

Reply:
ok, no need in that case